backend/app/assets/stylesheets/spree/backend/_bootstrap_custom.scss

```diff
@@ -68,6 +68,7 @@ $border-radius:          3px !default;
 $input-color:                    $color-3 !default;
 $input-border-color:             $color-border !default;
 $input-border-focus:             $color-2 !default;
+$input-color-placeholder:        lighten($gray-light, 20) !default;
```

Comment:
Cannot use `$gray-light` here since it's no longer defined until later on... this is tricky.